sdk/communication/communication-job-router/src/models/responses.ts

```diff
@@ -1,12 +1,33 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export {
-  AcceptJobOfferResponse,
-  CreateJobResponse,
-  UpsertQueueResponse,
-  UpsertChannelResponse,
-  UpsertClassificationPolicyResponse,
-  UpsertDistributionPolicyResponse,
-  UpsertExceptionPolicyResponse
-} from "../generated/src/models";
+import {
+  JobRouterCreateClassificationPolicyV2Response,
+  JobRouterPatchClassificationPolicyV2Response,
+  JobRouterCreateExceptionPolicyV2Response,
+  JobRouterPatchExceptionPolicyV2Response
+} from "../generated/src";
+
+/**
+ * Options to create a classification policy.
+ */
+export interface CreateClassificationPolicyResponse
```

Comment:
What's the purpose of these interfaces? Could we just expose the base type?

Reply:
Right, I will run an autorest update when [this task](https://skype.visualstudio.com/SPOOL/_workitems/edit/2693561) is done.

Reply:
Actually, these response and request are not from autorest, their purpose is to make the names consistent and short for end user.